app/javascript/app/components/ghg-emissions/ghg-emissions-selectors/ghg-emissions-selectors-data.js

```diff
@@ -503,8 +508,31 @@ export const getTableData = createSelector(
         {}
       )
     }));
+    return tableData;
+  }
+);
 
-    return pivot;
+export const getTitleLinks = createSelector(
+  [getTableData, getModelSelected, getRegions, getCountries],
+  (data, model, regions, countries) => {
```

Comment:
Both selectors `getRegions` and `getCountries` can return `null`, could you add guards for `regions` and `countries` here?